client/src/components/History/Content/Dataset/DatasetDetails.vue

```diff
@@ -17,9 +17,17 @@
                             result.genome_build
                         }}</b-link>
                     </span>
-                    <div v-if="result.misc_info" class="info">
-                        <span class="value">{{ result.misc_info }}</span>
-                    </div>
+                    <!-- Include Job Detail Provider and Dataset Error Details here -->
+                    <JobDetailsProvider
```

Comment:
@tcollins2011 We should only show tool.stderr if the state is in error right?  Check that first, then conditionally use the JobDetailsProvider to avoid extra requests here.